builtin/game/falling.lua

```diff
@@ -9,7 +37,7 @@ local builtin_shared = ...
 core.register_entity(":__builtin:falling_node", {
 	initial_properties = {
 		visual = "wielditem",
-		visual_size = {x = 0.667, y = 0.667},
+		visual_size = { x = SCALE, y = SCALE, z = SCALE },
```

Comment:
visual_size = {x = SCALE, y = SCALE, z = SCALE},
according to MT Code style guide